retrieval_service/sql-proxy/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM gcr.io/gcp-runtimes/ubuntu_20_0_4
```

Comment:
This Dockerfile needs to contain the code, tests, and dependencies. We will set a placeholder command but  in the Cloud Build we will use a different entrypoint
```suggestion
# Set python runtime
FROM python:3.11-slim

# Copy local code to the container image.
WORKDIR /app
COPY . ./

# Install dependencies.
RUN pip install --no-cache-dir -r requirements.txt -r requirements-test.txt

# Set placeholder command
CMD pytest datastore/providers/cloudsql_postgres_test.py

```

Reply:
We most likely need this file in the parent directory of retrieval service